apps/frontend/src/app/components/page-not-found/page-not-found.component.ts

```diff
@@ -0,0 +1,25 @@
+import { ChangeDetectionStrategy, Component, Inject, OnDestroy, OnInit } from '@angular/core'
+import { DOCUMENT } from '@angular/common'
+
+@Component({
+  selector: 'verseghy-page-not-found',
+  templateUrl: './page-not-found.component.html',
+  styleUrls: ['./page-not-found.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PageNotFoundComponent implements OnInit, OnDestroy {
+  private _noIndexElement: HTMLElement
+
+  constructor(@Inject(DOCUMENT) private _document) {}
+
+  ngOnInit(): void {
+    this._noIndexElement = document.createElement('meta')
```

Comment:
```suggestion
    this._noIndexElement = this._document.createElement('meta')
```

You shouldn't use the global document, use this._document instead